shrine.gemspec

```diff
@@ -51,10 +51,15 @@ direct uploads for fully asynchronous user experience.
   end
 
   gem.add_development_dependency "sequel"
-  gem.add_development_dependency "activerecord", "~> 4.2"
+
+  if RUBY_VERSION >= "2.3.0"
```

Comment:
Rails 5.x versions require MRI 2.2.2+, so maybe we could update the conditional to be `if RUBY_VERSION >= 2.2.2`?

In that case we can even loosen the conditional on ActiveRecord to be `~> 5.0`.

What do you think?